api.go

```diff
@@ -1460,3 +1460,14 @@ func ctxSetVersionInfo(r *http.Request, v *apidef.VersionInfo) {
 	}
 	setCtxValue(r, VersionData, v)
 }
+
+func ctxGetDefaultVersion(r *http.Request) bool {
+	if r.Context().Value(VersionDefault) != nil {
```

Comment:
semantically I would just do `return r.Context().Value(VersionDefault) != nil`

Reply:
Will do. Theres some debugging output i've left in there too so will clean that up too.